tests/python/test_logging.py

```diff
@@ -24,41 +24,51 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+import pytest
 from smartredis import *
 from smartredis.error import *
-import pytest
 
-@pytest.mark.parametrize("log_level", [
-    LLQuiet, LLInfo, LLDebug, LLDeveloper
-])
+
+@pytest.mark.parametrize("log_level", [LLQuiet, LLInfo, LLDebug, LLDeveloper])
 def test_logging_string(use_cluster, context, log_level):
-    log_data(context, log_level, f"This is data logged from a string ({log_level.name})")
-    log_warning(context, log_level, f"This is a warning logged from a string ({log_level.name})")
-    log_error(context, log_level, f"This is an error logged from a string ({log_level.name})")
+    log_data(
+        context, log_level, f"This is data logged from a string ({log_level.name})"
+    )
+    log_warning(
+        context, log_level, f"This is a warning logged from a string ({log_level.name})"
+    )
+    log_error(
+        context, log_level, f"This is an error logged from a string ({log_level.name})"
+    )
 
-@pytest.mark.parametrize("log_level", [
-    LLQuiet, LLInfo, LLDebug, LLDeveloper
-])
+
+@pytest.mark.parametrize("log_level", [LLQuiet, LLInfo, LLDebug, LLDeveloper])
 def test_logging_client(use_cluster, context, log_level):
     c = Client(None, use_cluster, logger_name=context)
     c.log_data(log_level, f"This is data logged from a client ({log_level.name})")
-    c.log_warning(log_level, f"This is a warning logged from a client ({log_level.name})")
+    c.log_warning(
+        log_level, f"This is a warning logged from a client ({log_level.name})"
+    )
```

Comment:
client and string are the same length.  I don't see why all the string ones above got split to multiple lines, but only some of the client ones did. Can we just keep all these on single lines?

Reply:
I added some column indicators at 80, 88, and 94 characters to illustrate:

![image](https://github.com/CrayLabs/SmartRedis/assets/3595025/dd23d9ac-377f-4bfa-bd8a-316c5e125d8c)

The `log_warning` calls are definitely over the 88 character limit. I can manually reformat this file for now. Without further tweaks, it would get picked up again on the next `make format`:

![image](https://github.com/CrayLabs/SmartRedis/assets/3595025/9140228e-bedd-4c4a-8992-21f245922776)

For now, I reformatted & disabled re-formatting in this file with `# fmt: off`